COPYING

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2003-2006 Eelco Dolstra
+Copyright (c) 2003-2006 Eelco Dolstra, Google Inc.
```

Comment:
@vcunat so is everyone ;-). It currently serves a "do not merge" function.
If #11557 merges this line will not be part of this PR.

On Fri, Dec 11, 2015 at 10:49 AM Vladimír Čunát <notifications@github.com>
wrote:

> In COPYING
> <https://github.com/NixOS/nixpkgs/pull/11535#discussion_r47368887>:
>
> > @@ -1,4 +1,4 @@
> > -Copyright (c) 2003-2006 Eelco Dolstra
> > +Copyright (c) 2003-2006 Eelco Dolstra, Google Inc.
>
> I'm certainly against adding that line.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/NixOS/nixpkgs/pull/11535/files#r47368887>.
>